sycl/test-e2e/Plugin/level_zero_dynamic_batch_test.cpp

```diff
@@ -1,6 +1,5 @@
 // REQUIRES: gpu, level_zero
-// TODO: Reenable, see https://github.com/intel/llvm/issues/14721
-// UNSUPPORTED: ze_debug, windows, linux
+// UNSUPPORTED: ze_debug
```

Comment:
Not blocking the merge, but should this eventually be enabled? If so, please, create another issue for this.

Reply:
I think it was always like that, even before PI removal.

Reply:
@steffenlarsen and/or @lbushi25 are working on an ongoing effort to review all `XFAIL` and `UNSUPPORTED` directives in our e2e tests to add an explicit comment that those are either expected to be there due to our implementation status/device capabilities, or that those should be fixed as part of this or that tracker.